{{cookiecutter.project_slug}}/.watchmanconfig

```diff
@@ -0,0 +1,13 @@
+{
+  "ignore_dirs": [
+    "node_modules",
+    ".envs",
+    "bin",
+    "lib",
+    "include",
+    ".git",
+    ".idea",
+    "templates",
```

Comment:
```suggestion
    "templates",
    "static",
```
I believe static files are also not included. The browser when testing caches them, sure as you can invalidate the browser cache of static files, but Django doesn't care, just like the templates dir.